lang_java/parsing/tree-sitter-parser.js

```diff
@@ -0,0 +1,62 @@
+const Parser = require('tree-sitter');
+const Java = require('../parsing/tree-sitter-java'); 
+
+var myArgs = process.argv.slice(2);
+
+var fs = require("fs");
+const sourceCode = fs.readFileSync(myArgs[0]).toString();
+
+const parser = new Parser();
+parser.setLanguage(Java); 
+const tree = parser.parse(sourceCode);
+
+var lines = sourceCode.split("\n");
+
+function get_chars(startRow, startCol, endRow, endCol) {
+    var result = "";
+    if (startRow == endRow) {
+        result = lines[startRow].substring(startCol, endCol);
+    } else {
+        result = lines[startRow].substring(startCol);
+        var k;
+        for (k = startRow+1; k < endRow; k++) {
+            result += lines[k];
+        }
+        result += lines[endRow].substring(0,endCol);
+    }
+    return result.trim();
+}
+
+function get_pos(startRow, startCol, endRow, endCol) {
+    var result = "\"pos\": [\"startrow\": " + startRow + ", \"startcol\": " + startCol + ", \"endrow\": " + endRow + ", \"endcol\": " + endCol +"]";
```

Comment:
Typically AST position data is in form of:
```
   "end": { "col": 29, "line": 3},
   "start": { "col": 1, "line": 1}
```
Babylon for Javascript for example has this format: https://astexplorer.net

Reply:
Ok, I can change it. That's actually what tree-sitter originally had